playground/frontend/lib/modules/messages/handlers/set_content_message_handler.dart

```diff
@@ -34,11 +37,21 @@ class SetContentMessageHandler extends AbstractMessageHandler {
       return MessageHandleResult.notHandled;
     }
 
-    _handle(message);
+    unawaited(_handle(message));
     return MessageHandleResult.handled;
   }
 
-  void _handle(SetContentMessage message) {
-    playgroundController.examplesLoader.load(message.descriptor);
+  Future<void> _handle(SetContentMessage message) async {
+    final descriptor = message.descriptor;
+
+    try {
+      await playgroundController.examplesLoader.load(descriptor);
+    } on Exception catch (ex) {
+      PlaygroundComponents.toastNotifier.addException(ex);
```

Comment:
`ExamplesLoader` is in `playground_components` package shared by Playground and Tour of Beam. This package allows for toasts but should not itself create ones because it will be hard to suppress them if Tour of Beam will ever require silent failures or its custom errors.